index.html

```diff
@@ -234,6 +226,35 @@ <h3>BrowserCaptureMediaStreamTrack extension</h3>
         </dl>
       </section>
     </section>
+    <section id="applying-the-restriction-transformation">
+      <h2><dfn>Applying the restriction transformation</dfn></h2>
+      <p>
+        When it's time to produce <var>frame</var> for a video track <var>videoTrack</var> that is
+        [=restricted=] to a given target <var>cropTarget</var>, execute the following algorithm:
+      </p>
+      <ol>
+        <li>
+          <!-- TODO(eladalon1983): Fix linkability of slots in the mediacapture-region spec. -->
+          If <var>cropTarget</var>.[[\Element]] does not represent an element that's in the DOM of
+          either the captured [=top-level browsing context=] or one of its descendant [=browsing
+          contexts=], abort without producing a new frame.
+        </li>
+        <li>
+          <!-- TODO(eladalon1983 / mfoltzgoogle): Improve spec language around rendering. -->
+          <!-- TODO(eladalon1983 / mfoltzgoogle): Specify the following "TODO reasons". -->
+          Because of TODO reasons, it is guaranteed that the <var>cropTarget</var>.[[\Element]] is
```

Comment:
The style preconditions listed in VT would apply to element capture:

https://drafts.csswg.org/css-view-transitions-1/#named-and-transitioning

Specifically:

> [Elements](https://drafts.csswg.org/css2/#element) [captured in a view transition](https://drafts.csswg.org/css-view-transitions-1/#captured-in-a-view-transition) during a [view transition](https://drafts.csswg.org/css-view-transitions-1/#view-transitions) or whose [view-transition-name](https://drafts.csswg.org/css-view-transitions-1/#propdef-view-transition-name) [computed value](https://www.w3.org/TR/css-cascade-5/#computed-value) is not [none](https://drafts.csswg.org/css-view-transitions-1/#valdef-view-transition-name-none) (at any time):
> Form a [stacking context](https://www.w3.org/TR/CSS21/visuren.html#x43).
> Are [flattened in 3D transforms](https://www.w3.org/TR/css-transforms-2/#grouping-property-values).
> Form a [backdrop root](https://drafts.fxtf.org/filter-effects-2/#backdrop-root).

These can be enforced by the browser internally applying certain constraints to the layout, which is what VT does.  I think that's the right approach here, until we have a concrete proposal to require a CSS property that would force these things to be true.